app/models/loan.rb

```diff
@@ -64,6 +70,8 @@ class Loan < ActiveRecord::Base
   attr_option_settable :status, :loan_type, :project_type, :public_level
 
 
+  attr_custom_model_linkable LOAN_CRITERIA_FIELD_SET_SYMBOL, POST_ANALYSIS_FIELD_SET_SYMBOL
```

Comment:
Yeah. I wanted to get feedback about best practice / style here.  The defined constants are being referenced by the unit test code, and I just realized should have also been referenced by the static_data.rb code.  There will also be UI code which needs to be specifically tied to those field set names.  I liked the idea guaranteeing that all those usages share the same symbol.  But I recognize this is a java pattern, and I was wondering if the "ruby way" is to just accept that changes to symbols can result in breakages and to rely on the unit tests to catch.

Do you prefer for me to remove the constants and just use direct symbol references?